### PR TITLE
London Class 7 - Benyam Worku - Git&GitHub - Week 1

### PR DESCRIPTION
**Pull request by BenyamWorku**
Made changes to some answers

# Your Details

- Your Name: Benyam Worku
- Your City: London
- Your Slack Name: Benyam Worku

# Homework Details

- Module: Git and Github
- Week: 2

### CODE REVIEW

**Comment by DanGhanbari** [comment]
Hi Benyam, 

I have reviewed your changes and it all looks correct to me.
The only thing you need to change is the name of your pull request. It should be in the format that the task says.
Well done!

Daniel

File: homework.md [hunk 1]
```diff
@@ -110,4 +113,4 @@ To record what changes you made to the code
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+The Git Cheatsheet! Stackoverflow etc.
```
Comment (alinosratipour) [inline homework.md:116]: i think  Git Documentation website as well.

**Review by alinosratipour — COMMENTED** [review]
I finished you're review

File: homework.md (hunk 1 above)
Comment (BenyamWorku) [reply homework.md:116]: Yep, I will add that one too. Thanks, alinosratipour

**Comment by BenyamWorku** [comment]
> Hi Benyam,
> 
> I have reviewed your changes and it all looks correct to me.
> The only thing you need to change is the name of your pull request. It should be in the format that the task says.
> Well done!
> 
> Daniel
Thanks Daniel. Sure I will change the name of the pull request.

**Pull request closed without merging**